lnwallet/wallet.go

```diff
@@ -907,6 +1004,77 @@ func (l *LightningWallet) handleContributionMsg(req *addContributionMsg) {
 			"generated: %v", chanPoint, spew.Sdump(fundingTx))
 	}
 
+	// This is a normal flow where no PSBT step is involved, so we can
```

Comment:
The comment says no PSBT step is involved, but the referenced method has PBST related logic.

Reply:
Fixed the comment.